src/modules/game/game.service.ts

```
@@ -1,20 +1,22 @@
+import { MakeMoveDto } from './dto/make-move.dto';
 import { throwHttpError } from 'src/common/errors/http-exception.helper';
 import { ErrorCode } from 'src/common/errors/error-codes.enum';
-import { MakeMoveDto } from './dto/make-move.dto';
 import { GameStatus, ResultReason, Winner } from 'src/shared/enum/game.enum';
 import { Chess } from 'chess.js';
-import { User } from 'src/schema/user.schema';
 import { AiService } from 'src/ai/ai.service';
 import { EventEmitter2 } from '@nestjs/event-emitter';
 import { GameGateway } from 'src/gateway/game.gateway';
+import { randomBytes } from 'crypto';
+import { NotFoundError } from 'rxjs';
```

ESLint

[error] 'NotFoundError' is defined but never used.

(@typescript-eslint/no-unused-vars)